pkg/osbuild/grub_iso_stage.go

```diff
@@ -23,8 +51,11 @@ type ISOKernel struct {
 
 // Assemble a file system tree for a bootable ISO
 func NewGrubISOStage(options *GrubISOStageOptions) *Stage {
+	if err := options.validate(); err != nil {
+		panic(err)
```

Comment:
Is this tested? Might be nice to add a tiny test (but given otk is coming maybe the rational is not to overinvest which is fine with me)

Reply:
It's tested in the sense that the tests were failing when it was added and the next commit fixes them :D 
I can add negative testing if you like.

Reply:
Added panic assertion tests both at the level of the stage itself (in `pkg/osbuild/`) and all the way up at the image serialization level.